soroban-env-host/src/native_contract/token/allowance.rs

```diff
@@ -37,7 +37,7 @@ pub fn write_allowance(
     // validates the expiration and then returns the ledger seq
     // The expiration can be less than ledger seq if clearing an allowance
     let ledger_seq = e.with_ledger_info(|li| {
-        if expiration > e.max_expiration_ledger()? {
+        if expiration > e.max_live_until_ledger()? {
```

Comment:
Rename `expiration` and remove from comment.

Reply:
Fixed